projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-universal.php

```diff
@@ -368,6 +417,87 @@ public function remove_from_cart_via_quantity() {
 		);
 	}
 
+	/**
+	 * Tracks any additional blocks loaded on the Cart/Checkout page.
+	 *
+	 * @var array
+	 */
+	private $additional_blocks_on_page = array();
+
+	/**
+	 * Gets the inner blocks of a block.
+	 *
+	 * @param array $inner_blocks The inner blocks.
+	 *
+	 * @return array
+	 */
+	private function get_inner_blocks( $inner_blocks ) {
+		$block_names = array();
+		if ( $inner_blocks['blockName'] ) {
+			$block_names[] = $inner_blocks['blockName'];
+		}
+		if ( $inner_blocks['innerBlocks'] ) {
+			$block_names = array_merge( $block_names, $this->get_inner_blocks( $inner_blocks['innerBlocks'] ) );
+		}
+		return $block_names;
+	}
+
+	/**
+	 * Gets the IDs of additional blocks on the Cart/Checkout pages or templates.
+	 *
+	 * @return void
+	 */
+	public function get_additional_blocks_on_page() {
+		global $post;
+		$content = $post->post_content;
+
+		$cart_checkout_templates_in_use = class_exists( 'Automattic\WooCommerce\Blocks\Package' ) && version_compare( Automattic\WooCommerce\Blocks\Package::get_version(), '10.6.0', '>=' );
+
+		if ( $cart_checkout_templates_in_use ) {
+			if ( function_exists( 'get_block_template' ) ) {
+				$content = get_block_template( 'woocommerce/woocommerce//page-checkout' )->content;
+			}
+
+			if ( function_exists( 'gutenberg_get_block_template' ) ) {
+				$content = gutenberg_get_block_template( 'woocommerce/woocommerce//page-checkout' )->content;
```

Comment:
The double slashes here are done on purpose?

Reply:
Yep, thanks for checking :)